src/hooks/use-swr-document.ts

```diff
@@ -342,9 +361,12 @@ export const useDocument = <
         })
       }
       if (!path) return null
-      return fuego.db.doc(path).set(data, options)
+      let docRef = fuego.db.doc(path)
+      if (documentDataConverter)
```

Comment:
I already asked this above, but flagging here too. Should the converter be applied to firestore refs that are being added, or is it only for reading in data?